CMakePresets.json

```diff
@@ -0,0 +1,125 @@
+{
+  "version": 4,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 23,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "nexstim",
+      "generator": "Visual Studio 16 2019",
+      "binaryDir": "../BUILD-3.6.7",
+      "cacheVariables": {
+        "BUILD_APPS": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "BUILD_SHARED_LIBS": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "BUILD_SINGLE_SHARED_LIBRARY": {
+          "type": "BOOL",
+          "value": "OFF"
+        },
+        "CMAKE_CONFIGURATION_TYPES": {
+          "type": "STRING",
+          "value": "Debug;Release;MinSizeRel;RelWithDebInfo;UsableDebug"
+        },
+        "CMAKE_CXX_FLAGS": {
+          "type": "STRING",
+          "value": "/Zi /MP /W3 /Gy /Zc:wchar_t /Gm- /fp:precise /errorReport:none /GF /WX- /Zc:forScope /Gd /EHa /nologo /DNOMINMAX /DNO_WARN_MBCS_MFC_DEPRECATION /D_CRT_SECURE_NO_DEPRECATE /D_SCL_SECURE_NO_WARNINGS /DWIN32 /D_MBCS /DWINVER=0x0A00 /Zi /DQT_MESSAGELOGCONTEXT"
+        },
+        "CMAKE_CXX_FLAGS_DEBUG": {
+          "type": "STRING",
+          "value": "/MDd /RTCsu /Od /Ob0 /GS /D_DEBUG /DNBS_ENABLE_MEMORY_LEAK_DETECTION"
+        },
+        "CMAKE_CXX_FLAGS_MINSIZEREL": {
+          "type": "STRING",
+          "value": "/MD /O1 /Os /Oy /Ob2 /Gs /GL /DNDEBUG /Zi"

```

Comment:
Remove all /GL (Whole program optimization) flags. This prevents using library in different versions of the visual studio (see "ITK Build HowTo.txt" in https://svn.nexstim.local/svn/ExternalLibs/ITK/ITK-5_0_1/MSVC142_X64 and https://jira.nexstim.local:8443/browse/SNBT-233).